lib/review/plaintextbuilder.rb

```diff
@@ -723,6 +723,14 @@ def nofunc_text(str)
       str
     end
 
+    def graph_mermaid(_id, file_path, _line, _tf_path)
+      file_path
```

Comment:
```suggestion
if self.insntace_of?(ReVIEW::PLAINTEXTBuilder)
  file_path
else
  super
end
```

こちらをこう変えて、TOPBuidlerでは何もしないイメージです。
（Builder#graph_mermaidでこういう分岐をしてもいいはずですが、こちらのクラスに書いた方が何かあったときに探しやすいかな…ということで。）

Reply:
ふぉー、すごい…